src/settings.cpp

```diff
@@ -533,6 +533,13 @@ if ( GetFlagIniSet ( IniXMLDocument, "client", "defcentservaddr", bValue ) )
         bWindowWasShownConnect = bValue;
     }
 
+    // selected Settings Tab
+    if ( GetNumericIniSet ( IniXMLDocument, "client", "settingstab",
+         0, 3, iValue ) )
```

Comment:
Really minor and haven't tested it, but if `0, 3` is the inclusive range of permitted values and  there are only three tabs, then I think this should rather be `0, 2` to permit indices 0, 1, and 2?